rfcs/2021-07-19-8216-multiple-pipelines.md

```diff
@@ -257,6 +254,10 @@ Doesn't block to create other sources/sinks.
 
 ## Outstanding Questions
 
+1. Will components defined in the top-level configuration be considered part of a "default" pipeline or no pipeline at all? Sources and sinks seems straightforward to leave out of pipelines, but transforms are a more interesting question. They're the most obvious way to implement filters that limit the data available to specific pipelines.
+1. Is the concept of routes worth introducing, or should we allow an `outputs` option on transforms within a pipeline instead? Routes as described here are simple output connections, quite different from the same concept from competitors where they act as input definitions. If we think we'll want to add that later it may be worth saving the name.
+1. Do we need to consider pipelines with no transforms? As described, they could just be a route, but there would be no actual component for collecting metrics and therefore not observable.
```

Comment:
I agree, they are not really interesting but the question "what would an empty pipeline do" was raised, so I preferred adding it to the RFC.

Reply:
Yes makes sense. I think the question is mostly about the observability component, since with no transforms there would be not metrics. But this may also tie into the discussion of `routes` below. If routes exist as their own thing, a pipeline file could exist that only has routes and demonstrates that lack of metrics. If we instead just have `outputs` on transforms, then it's impossible to define an empty pipeline.

Reply:
I updated the RFC to use `outputs` instead of `Routes`, so I guess we can consider this closed?